community/earning.md

```diff
@@ -4,30 +4,68 @@ description: Learn how community members get rewarded by contributing to AirSwap
 
 ## Circle Funding
 
-The contributor circle is funded from two sources - a base level of funding distributed from the treasury, and a bonus funding pool derived from trade fees collected. The base fundings are distributed in AST while the bonus fundings are distributed in the currency of the consolidated token pool (currently WETH).
+The contributor circle is funded by a portion of the monthly trade fees collected (currently 14%). AST from the Treasury will be used to top up Contributor tips to the base level if there is a shortfall in the amount of fees collected for the month. 
```

Comment:
Contributor circles are rewarded by a percentage of protocol fees (14% at time of writing). In case of a shortfall against the monthly base amount (below), AST is withdrawn from `treasury.airswap.eth` to ensure the base amount is met and distributed.